src/db-transaction/db-transaction.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable, OnModuleDestroy } from '@nestjs/common';
+import { DataSource, EntityManager, QueryRunner } from 'typeorm';
+import { IsolationLevel } from 'typeorm/driver/types/IsolationLevel';
+
+@Injectable()
+export class DbTransactionService implements OnModuleDestroy {
+    private readonly queryRunnerSet: Set<QueryRunner>;
```

Comment:
In the advent that `onModuleDestroy` function is not needed as suggested in  a comment below , i believe this variable existence would be futile